tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -7,36 +7,57 @@
 
 
 @pytest.mark.parametrize(
-    "number, currency, expected",
+    "value, currency, decimal_limit, formatted_currency",
     (
-        ("11", "GBP", "£11.00"),
-        ("11.99", "GBP", "£11.99"),
-        ("11000", "USD", "US$11,000.00"),
-        (0, None, "£0.00"),
-        (0.00, None, "£0.00"),
+        (2, "GBP", 6, "£2.00"),
+        ("11", "GBP", 2, "£11.00"),
+        ("11.99", "GBP", 2, "£11.99"),
+        ("11000", "USD", 2, "US$11,000.00"),
+        (0, None, 2, "£0.00"),
+        (0.00, None, 2, "£0.00"),
```

Comment:
Why are these set to None?. We are defaulting to GBP in the test body but why?.

Reply:
I have removed these as not sure either why GBP was being defaulted in the test (have also removed that) 👍